corehq/apps/locations/tests/test_bulk_management.py

```diff
@@ -875,9 +879,9 @@ def test_custom_data(self):
         self.assertLocationTypesMatch(FLAT_LOCATION_TYPES)
         self.assertLocationsMatch(self.as_pairs(tree))
 
-        locations = SQLLocation.objects.all()
-        self.assertEqual(locations[0].metadata, {'a': '1'})  # test that ints are coerced to strings
-        self.assertEqual(locations[1].metadata, {'国际字幕': '试验'})
+        locations = _codify(SQLLocation.objects.all())
+        self.assertEqual(locations["s1"].metadata, {'a': '1'})  # test that ints are coerced to strings
+        self.assertEqual(locations["c1"].metadata, {'国际字幕': '试验'})
```

Comment:
Wow yeah, that was ugly.  I assume you've seen `LocationHierarchyPerTest` and the associated `setup_locations_and_types` utils?  What you did here seems easier than refactoring to use that, but it's an option if you get frustrated with stuff like this in the future.